tests/helpers_tests/test_condition.py

```diff
@@ -1206,15 +1206,15 @@ def test_invalid_semver__returns_None__when_semver_is_invalid(self):
             self.assertIsNone(result, custom_err_msg)
 
     def test_qualified__returns_true__when_user_is_qualified(self, ):
-        self.user_context.qualified_segments = ['odp-segment-2']
+        self.user_context._qualified_segments = ['odp-segment-2']
         evaluator = condition_helper.CustomAttributeConditionEvaluator(
             qualified_condition_list, self.user_context, self.mock_client_logger,
         )
 
         self.assertStrictTrue(evaluator.evaluate(0))
 
     def test_qualified__returns_false__when_user_is_not_qualified(self, ):
-        self.user_context.qualified_segments = ['odp-segment-1']
+        self.user_context._qualified_segments = ['odp-segment-1']
```

Comment:
Don't we need support for atomic getter for "qualified_segments"?  like "get_user_attributes()"

Reply:
My bad, I thought we were going to implement those during a subsequent PR. I've added a getter and setter for qualified segments.